src/Compilers/Core/Portable/Syntax/SeparatedSyntaxList.cs

```diff
@@ -573,5 +573,15 @@ IEnumerator IEnumerable.GetEnumerator()
 
             return SpecializedCollections.EmptyEnumerator<TNode>();
         }
+
+        public static implicit operator SeparatedSyntaxList<SyntaxNode>(SeparatedSyntaxList<TNode> nodes)
```

Comment:
@dotnet/roslyn-compiler Please take a look at this.  Current ```SyntaxList<T>``` has these operators, but we missed them for SeparatedSyntaxList.  Anyone have a problem with us adding this?